TEKDB/TEKDB/admin.py

```diff
@@ -965,10 +972,10 @@ class PlacesAdmin(NestedRecordAdminProxy, RecordModelAdmin):
     change_list_template = "admin/TEKDB/places/change_list.html"
 
     def changelist_view(self, request, extra_context=None):
-        from .views import getPlacesGeoJSON
+        from .views import get_places_geojson
```

Comment:
I renamed the function to snake case instead of camel case to follow convention.